test/constructors.jl

```diff
@@ -97,9 +97,9 @@ const ≅ = isequal
     @test df[!, "x1"] == df2[!, "x1"]
     @test df[!, "x2"] == df2[!, "x2"]
 
-    @test_throws ArgumentError DataFrame!([0.0 1.0;
-                                           0.0 1.0;
-                                           0.0 1.0])
+    @test_throws MethodError DataFrame([0.0 1.0;
```

Comment:
Why `MethodError`? :-/

Reply:
because when converting form a matrix we do not allow `copycols` kwarg. Do you think we should add it (it would be ignored).

Reply:
Ah OK that's fine.